client/src/components/Related/CompareWindow/index.jsx

```diff
@@ -3,26 +3,24 @@ import { useState, useEffect } from "react";
 import styled from "styled-components";
 import OverlayWindow from "/client/src/components/assets/OverlayWindow.jsx";
 import ActionBtn from "../ActionBtn/ActionBtn.jsx";
+import Table from "./table.jsx";
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { solid, thin, regular, brands, icon } from '@fortawesome/fontawesome-svg-core/import.macro';
 
-const CompareItems = ({item, list, setList}) => {
+const CompareItems = ({item, product}) => {
 
   const [modal, setModal] = useState(false);
 	const toggleModal = () => {setModal((value) => (!value))}
 
-  const handleClick = () => {
-
-  };
-
-  return (
+    return (
     <>
       <ActionBtn onClick={toggleModal}>
         <FontAwesomeIcon icon={regular("star")} />
       </ActionBtn>
       {modal ?
```

Comment:
It's always kinda weird how the modal windows just sort-of hang out near the things that spawn them in the DOM, isn't it? Makes doing those <></> wrappers necessary.

Reply:
any changes necessary here

Reply:
Nah this one's fine, just me griping about react.